content/millennium/r4/base/workflow/appointment.md

```diff
@@ -69,6 +70,7 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
 
 ID                               | Value\[x] Type                                              | Description
 -----------------------------    |-------------------------------------------------------------|--------------------------------------------------------------------
+`action-comment`                 | [`string`](https://hl7.org/fhir/r4/datatypes.html#string)   | The comment to be associated with the action performed on the resource.
```

Comment:
can we list these in alphabetical order?

Reply:
Addressed in 4205ae8ab1907346bf0f704e3fd9134667ac0d9d